docs/src/components/productX/XHero.tsx

```diff
@@ -174,13 +276,28 @@ export default function XHero() {
                   }),
               ]}
             >
-              <DataGridPro
+              <DataGridPremium
```

Comment:
@joserodolfofreitas I've submitted a follow-up PR for this demo: https://github.com/mui/material-ui/pull/37001